include/proper_common.hrl

```diff
@@ -34,8 +34,7 @@
 -define(FORALL(X,RawType,Prop), proper:forall(RawType, fun(X) -> Prop end)).
 -define(EXISTS(X,RawType,Prop), proper:exists(RawType, fun(X) -> Prop end, false)).
 -define(NOT_EXISTS(X,RawType,Prop), proper:exists(RawType, fun(X) -> Prop end, true)).
--define(FORALL_TARGETED(X, RawType, Prop),
-        proper:targeted(RawType, fun(X) -> Prop end)).
```

Comment:
I think this was written this way because we do not want lines > 80 chars

Reply:
Yes will update with fixes as well.